frontend/views.py

```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.http import HttpResponse, HttpResponseNotFound
+
+
+# Create your views here. Class based views, we scrap the idea for now and go for function based views.
+""" def base_view(request):
```

Comment:
Uncomment and use these instead of the TemplateView you're currently using

Reply:
sounds good, me fix